symfony/src/Repository/ArticleRepository.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\Article;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+
+/**
+ * @method Article|null find($id, $lockMode = null, $lockVersion = null)
+ * @method Article|null findOneBy(array $criteria, array $orderBy = null)
+ * @method Article[]    findAll()
+ * @method Article[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class ArticleRepository extends ServiceEntityRepository
+{
+    public function __construct(RegistryInterface $registry)
+    {
+        parent::__construct($registry, Article::class);
+    }
+
+    // /**
+    //  * @return Article2[] Returns an array of Article2 objects
+    //  */
+    /*
+    public function findByExampleField($value)
```

Comment:
Is this code commented on purpose?

Reply:
Same here, i don't understand this comments here buddy.

Reply:
Yes, it's commented by default, i use "symfony maker bundle", it help the user create entity / controller etc .. I've created my Entity Article with this command and it generate me the RepositoryClass for my Article Entity. 
But there is a mistake on the comment thank you. 
This should be `@return Article[] Returns an array of Article objects.`

Reply:
Wouldn't it be nice to have one or two real methods like `findOneById` or `findOneByAuthor` instead of commented ones? .. WDYT @nunomaduro @TomKlotzPro ?